feature-detects/css/flexgap.js

```diff
@@ -7,26 +7,27 @@
   "notes": [{
     "name": "W3C Spec",
     "href": "https://www.w3.org/TR/css-align-3/#gaps"
-  }]
+  }],
+  "authors": "Chris Smith (@chris13524)"
```

Comment:
Authors has to be an array. That would have shown up on the travis tests if there wasnt a bug with my code :-) Fixed that so if you rebase now against master you wil lsee that the tests fail :-)

Reply:
D'oh!